test/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCoreTests/EntityFrameworkDiagnosticListenerTests.cs

```diff
@@ -63,7 +64,7 @@ public void EntityFrameworkContextEventsInstrumentedTest()
                 Assert.Equal("ItemTwo", items[2].Name);
             }
 
-            Assert.Equal(2, activityProcessor.Invocations.Count);
+            Assert.Equal(3, activityProcessor.Invocations.Count);
```

Comment:
OpenTelemetry 1.x additionally invokes `SetParentProvider` on each test. So, I've increased this value